ntbs-service/Pages/Shared/_Form_Layout.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@{
+    Layout = "./_Layout.cshtml";
+}
+<form method="post" autocomplete="off">
+    <nhs-grid-row style="position: relative">
```

Comment:
Looks like a copy and paste, but do you know if the relative is required (for either here or in the original base Layout)?